### PR TITLE
docs: adds contributing guide & updates issue template

### PR DESCRIPTION
**Pull request by tcasey**
## Checklist
- [x] Atomic commits
- [x] Tests adjusted to address changes
- [x] Documentation
- [x] Storybook adjusted to reflect changes (if applicable)
- [x] Reviewed by author
- [x] Ready for review

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+* Open a new Gitub pull request with the patch.
+* The PR template will outline all of the questions that a contributor should ask themself before asking a codeowner to review their PR for approval
+* Check off each item in the PR template checklist whether it applies or not as a sign that you've considered the question and complied appropriately for your contribution
+* If you want preliminary review of your ideas, feel free to slack a code owner, too.
+* When you're ready for a formal review, slack a code owner
+
+## Do you have questions
+
+* Ask them. Slack a code owner
```
Comment (aFluttr) [inline .github/CONTRIBUTING.md]: Who is them?

File: README.md [hunk 2]
```diff
@@ -36,6 +50,6 @@ We have a `postinstall` that runs [rndebbugger-open](https://www.npmjs.com/packa
 
 ## Published Storybook
 
-You can find the expo community app [here](https://expo.io/@vivintsolar/vslr-ui-kit) or if you already have [expo](https://expo.io/) installed on your phone then scan the QR code down below 👇🏼
+You can find the expo community app [here](https://expo.io/@vivintsolar/vslr-ui-kit) or if you already have [expo](https://expo.io/) installed on an android phone then scan the QR code down below 👇🏼
```
Comment (aFluttr) [inline README.md:53]: What about iOS?

Comment (tcasey) [reply README.md:53]: iOS can't use the qr code

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (tcasey) [reply .github/CONTRIBUTING.md]: Forgot to add the code owners, I'll add the link

File: README.md (hunk 2 above)
Comment (aFluttr) [reply README.md:53]: Maybe mention that in there

**Review by aFluttr — APPROVED** [review]

**Pull request merged**